apps/connect/src/components/modal/modals/AddDriveModal.tsx

```diff
@@ -16,7 +17,7 @@ type Props = {
 export function AddDriveModal(props: Props) {
     const { open, onAddLocalDrive, onAddRemoteDrive, onClose } = props;
     const user = useUser();
-    const apps = useApps();
```

Comment:
what we have been calling apps are just a special case of editor. they use the exact same code, but have a known documentType of `powerhouse/document-drive`. 

There is no reason for us to have a whole set of special code for handling this case, and calling these drive editors "apps" in our source code is needlessly confusing.